### DISCUSSION TITLE
How can I style svg icon in chatButton.

### DISCUSSION THREAD

**Discussion by Xiddiqui** [tjtanjin/react-chatbotify#67] [General]
Hello . I am facing issue to set height and width of my svg icon . inside options 
chatButton: {
            icon: "/assets/images/msg.svg",
        },
chatButtonStyle: {
            height: "40px",
            width: "40px",
            borderRadius: "20px",
        },
In chat button style when i pass height and width it apply on button i want to adjust my icon inside that button . Also when i pass background color in chatButton style my icon hides

**Comment by tjtanjin** [comment]
The svg icon is currently using `background` property. If you're adding color then you'll overwrite it with `background` as well. For the `chatButtonStyle`, it is intended to target the button. Is there something specific you're trying to achieve by resizing the icon but not the button? The icon would either have empty spaces around it or be too large for the button if you do that. If you really wish to one way is to modify and provide an icon directly via `chatButton`.

**Reply by Xiddiqui** [reply]
Okay thank you @tjtanjin

**Reply by tjtanjin** [reply]
> Okay thank you @tjtanjin 

No problem, let me know if you need more help 😊
